Flutter/datagrid/placeholder.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Placeholder in Flutter DataGrid | DataTable | Syncfusion
+description: Learn here all about how to add a placeholder to the Syncfusion Flutter DataGrid (SfDataGrid) control and more.
+platform: flutter
+control: SfDataGrid
+documentation: ug
+---
+
+# Placeholder in Flutter Datagrid (SfDataGrid)
+
+The DataGrid provides built-in support for displaying a placeholder when the row count is zero by setting the [SfDataGrid.placeholder]() property. When the `SfDataGrid.placeholder` is set, the DataGrid automatically shows the specified widget when there are no rows.
+
+By default, the DataGrid does not display anything when there are no rows. However, when a `placeholder` is applied, the DataGrid will display the corresponding widget specified in the `placeholder` property.
+
+The following example shows how to add a `placeholder` in DataGrid.
```

Comment:
The `SfDataGrid` provides built-in support for displaying a placeholder when the data source is empty by setting the [SfDataGrid.placeholder]() property. When the `SfDataGrid.placeholder` is set, the DataGrid automatically shows the specified widget in the scroll view area. By default, the `SfDataGrid` does not display anything when the data source is empty. 

The following example shows how to add a `placeholder` in `SfDataGrid`: